stock_scanner_inventory/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Please use newer header, although I ask whether this is necessary at all in a blank file?